sw/device/tests/sim_dv/all_escalation_resets_test.c

```diff
@@ -359,6 +359,11 @@ static void otbn_ctrl_fault_checker(bool enable) {
 static void otp_ctrl_fault_checker(bool enable, const char *ip_inst,
                                    const char *type) {
   // Check the otp_ctrl integrity fatal error code.
+  if (type != we_check) {
```

Comment:
I would replace this by a `CHECK` macro.

Reply:
Thanks for the suggestion, updated.